### PR TITLE
Suggest GitHub issue templates for recipe-scrapers library

### LINKED ISSUE #146: Create a GitHub issue template for new scraper requests
See https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Include:

- Advice that recipes should be public
- Minimum set of contents required
- Example recipe URL

Mention:

- Microformats are supported
- Pull requests are welcome
- Example pull request URL

### PR DESCRIPTION
**Pull request by jayaddison**
It's a little tricky to test these before they're merged into to the GitHub repo, as far as I know - and maybe the issue report volume is still low enough that we don't really need them yet.  But I figured they might be helpful.  What do you think about the idea and tone of these, @bfcarpio @hhursev? 

Here are a couple of references from GitHub's documentation about issue / PR templates:

- https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates
- https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Resolves #146

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/scraper_bug_report.md
Comment (bfcarpio) [inline .github/ISSUE_TEMPLATE/scraper_bug_report.md:16]: I'm not sure what our major mode of operation should be on this. 

On one hand we're not backporting fixes to previous versions, and we've only had one major breaking change with Python versions (#188 ) so "update to latest version" might be the best way of ensuring consistency. Updating the version might be something to put on the "do this before filing bug" procedure.

On the other most of our business logic doesn't change with version updates. A bug in version 8 is probably still around in version 10 etc.

File: .github/ISSUE_TEMPLATE/scraper_bug_report.md
Comment (bfcarpio) [inline .github/ISSUE_TEMPLATE/scraper_bug_report.md:11]: Can we ask for Python version and OS? We recently had an issue that was potentially OS specific and it seems that all 3 of us maintainers run some sort of *nix.

File: .github/ISSUE_TEMPLATE/scraper_bug_report.md
Comment (bfcarpio) [inline .github/ISSUE_TEMPLATE/scraper_bug_report.md:10]: Maybe we should specify a website not being supported is not a bug, but should be instead handled with the other template.

**Review by bfcarpio — COMMENTED** [review]
I was just thinking about doing this over the weekend =P

Just some nitpicks for you. Specifying Python version and OS are probably the only things I have a strong opinion on.

File: .github/ISSUE_TEMPLATE/new_scraper.md
Comment (jayaddison) [reply .github/ISSUE_TEMPLATE/new_scraper.md:16]: Good question - to explain a bit, initially I thought about not having a third checkbox at all.  But then I wondered whether people might not fully read or comprehend the prompt, and instead be drawn straight to the 'interactive' options -- and so I figured it might be worth adding one to allow/try to clear up any misunderstandings there by providing an option.

Unfortunately markdown doesn't seem to support radio buttons as far as I know, so your example may indeed happen :)

I think in that case we'd probably want to ask the issue author when they might like to get started working on it, and based on that we could decide whether we'd be likely to implement it ourselves (based on availability and what the site/schema look like) or allow them to try it themselves.

If we ask that it might turn out it was an accidental double-selection, or we could discuss the timeline (without providing any guarantees of course).

(that's a lot of semantics/context to describe in the file -- so perhaps there's a better way to phrase the options and/or prompt to help with all that - ideas welcome!)

File: .github/ISSUE_TEMPLATE/scraper_bug_report.md
Comment (jayaddison) [reply .github/ISSUE_TEMPLATE/scraper_bug_report.md:16]: That's a pretty good idea, I think.  There's definitely ambiguity in whether the author would expect a fix to land for their version (without any other changes), and whether there's any duty for them to try to check for bugfixes and/or verify in the latest version (which we probably should, given that it's relatively quick and can save everyone a lot of time).

I'll apply some changes here soon.

File: .github/ISSUE_TEMPLATE/new_scraper.md
Comment (jayaddison) [reply .github/ISSUE_TEMPLATE/new_scraper.md:16]: NB: Also potentially relevant context: I do think that (at least as currently written), ticking boxes one and two is a valid combination.

Comment (bfcarpio) [reply .github/ISSUE_TEMPLATE/new_scraper.md:16]: I agree that ticking boxes 1 and 2 at the same time. I'm not worried about this and I'm pretty sure this gets into deep lore about the English language. Not a big issue.

**Review by bfcarpio — APPROVED** [review]
These templates are very nice and I hope we'll see them in use soon!

**Comment by hhursev** [comment]
😍  party on @jayaddison 👍 👍  Feel free to do whatever you need after the merge too - it's OK with me. I approve the idea and I appreciate your effort. Thanks!

**Comment by jayaddison** [comment]
Thanks @bfcarpio @hhursev :)  Let's give this a try..

**Pull request merged**
